krcal/core/map_functions.py

```diff
@@ -218,8 +218,7 @@ def amap_replace_nan_by_mean(amap : ASectorMap, amMean : FitMapValue)->ASectorMa
                       lt    = amap.lt.copy().fillna(amMean.lt),
                       e0u   = amap.e0u.copy().fillna(amMean.e0u),
                       ltu   = amap.ltu.copy().fillna(amMean.ltu),
-                      mapinfo   = None)
-
+                      mapinfo   = amap.mapinfo)
```

Comment:
This should probably calculate mean from the map and then replace nan with that value